src/app/api/api/api.py

```diff
@@ -0,0 +1,9 @@
+from fastapi import APIRouter
+
+from .endpoints import oauth, user, login
+
+api_router = APIRouter()
+
+api_router.include_router(login.router, tags=["login"])
```

Comment:
router의 내부에 모두 /login이 포함되어 있다면, prefix에 login을 넣어보는건 어떨까요 ?
```suggestion
api_router.include_router(login.router, prefix="/login", tags=["login"])
```

Reply:
네네 수정하겠습니다!!